cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -306,6 +333,67 @@ message AvailablePackageDetail {
   google.protobuf.Any custom_detail = 13;
 }
 
+// InstalledPackageSummary
+//
+// An InstalledPackageSummary provides a summary of an installed package
+// useful when aggregating many installed packages.
+message InstalledPackageSummary {
+  // TODO: add example for API docs
+  // option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
+  //   example: '{}'
+  // };
+
+  // InstalledPackageReference
+  //
+  // A reference uniquely identifying the package.
+  InstalledPackageReference installed_package_ref = 1;
+
+  // Name
+  //
+  // A name given to the installation of the package (eg. "my-postgresql-for-testing").
+  string name = 2;
+
+  // PkgVersionReference
+  //
+  // The package version reference defines a version or constraint limiting
+  // matching package versions.
+  VersionReference pkg_version_reference = 3;
+
+  // CurrentPkgVersion
+  //
+  // The version of the package which is currently installed.
+  string current_pkg_version = 4;
+
+  // Installed package icon URL
+  //
+  // A url for an icon.
+  string icon_url = 5;
```

Comment:
fluxv2 HelmRelease doesn't have a field for this. Would the plug-in leave that empty or have some logic to derive it from available package?

Reply:
Yes the latter. The client (such as Kubeapps dashboard) wants to present to the user all the installed apps, including the icons (which come from the package, or in your specific case, the `HelmChart`). Similar for some other fields. Earlier on we looked at whether we wanted to instead include the available package nested in the installed package, but decided to use redundant fields on the installed package.